x-pack/plugins/fleet/server/services/output.ts

```diff
@@ -356,7 +356,7 @@ class OutputService {
           esClient,
           defaultDataOutputId,
           { is_default: false },
-          { fromPreconfiguration: options?.fromPreconfiguration ?? false }
+          { fromPreconfiguration: options?.fromPreconfiguration ?? false, asHelper: true }
```

Comment:
@nchaulet It would be nice to add some explaining comment on why we are updating the output from `is_default:true` to `false` here, it is not very obvious.

Reply:
the is_default flag act as a toggle enabling a new default disable the old one, we can probably wrap that code block in a function like `toggleDefaultOutput` to make this more explicit